DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsControllerTests.cs

```diff
@@ -0,0 +1,153 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.TrackingSystem.CentreConfiguration
+{
+    using System.Security.Claims;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using DigitalLearningSolutions.Web.Controllers.TrackingSystem;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.AspNetCore.Mvc;
+    using FluentAssertions.Execution;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using NUnit.Framework;
+
+    public class RegistrationPromptsControllerTests
+    {
+        private ICentresDataService centresDataService;
+        private ICustomPromptsService customPromptsService;
+        private CentreConfigurationController centreConfigurationController;
+        private ISession httpContextSession;
+        private const int CentreId = 2;
+
+        [SetUp]
+        public void Setup()
+        {
+            centresDataService = A.Fake<ICentresDataService>();
+            customPromptsService = A.Fake<ICustomPromptsService>();
+
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new[]
+            {
+                new Claim("UserCentreID", CentreId.ToString())
+            }, "mock"));
+            httpContextSession = new MockHttpContextSession();
+
+            centreConfigurationController = new CentreConfigurationController(centresDataService, customPromptsService)

```

Comment:
Dan made some extensions for this a while back so you should be able to do a neater setup.
See ResetPasswordControllerTests for an example (setup's something like; new Controller().WithDefaultContext().WithMockUser(true))